### PR TITLE
Extra datum

### PR DESCRIPTION
**Pull request by lisicky**

### CODE REVIEW

**Review by lisicky — COMMENTED** [review]
/check

File: rust/pkg/cardano_serialization_lib.js.flow [hunk 1]
```diff
@@ -8561,6 +8561,16 @@ declare export class TransactionBuilder {
     output_builder: TransactionOutputAmountBuilder
   ): void;
 
+  /**
+   * @param {PlutusData} datum
+   */
+  add_extra_datum(datum: PlutusData): void;
```
Comment (vsubhuman) [inline rust/pkg/cardano_serialization_lib.js.flow:8567]: Can it be renamed into `add_extra_witness_datum` and `get_extra_witness_datums`?

Comment (lisicky) [reply rust/pkg/cardano_serialization_lib.js.flow]: Done

**Review by vsubhuman — APPROVED** [review]

**Review by lisicky — COMMENTED** [review]
/check

**Pull request merged**